trailblazer/store/crud/update.py

```diff
@@ -116,6 +116,20 @@ def cancel_ongoing_analysis(
         )
         self.commit()
 
+    def cancel_slurm_analysis(
+        self, analysis: Analysis, analysis_host: Optional[str] = None
+    ) -> None:
+        """Cancel slurm analysis by cancelling all associated slurm jobs."""
+        for job in analysis.jobs:
+            if job.status in SlurmJobStatus.ongoing_statuses():
+                LOG.info(f"Cancelling job {job.slurm_id} - {job.name}")
+                cancel_slurm_job(analysis_host=analysis_host, slurm_id=job.slurm_id)
+
+    def cancel_tower_analysis(self, analysis: Analysis) -> None:
+        """Cancel a NF-Tower analysis."""
+        LOG.info(f"Cancelling Tower workflow for {analysis.family}")
+        self.query_tower(config_file=analysis.config_path, case_id=analysis.family).cancel
```

Comment:
Will Tower cancel associated Slurm jobs?

Reply:
If so add it to doc str

Reply:
I think it does. But I will test and make sure before resolving this

Reply:
It does. I'll update the docstring

Reply:
done